packages/account/src/Sections/Security/Passkeys/components/passkey-removing-verification.tsx

```diff
@@ -0,0 +1,22 @@
+import { Localize } from '@deriv-com/translations';
+// import { mobileOSDetect } from '@deriv/shared';
```

Comment:
do we  need this?

Reply:
@amina-deriv , all stuff in removing verification component is not used. it is just starting point to future implementation. 
The logic should be checked and refactored as well as the content